openc3/lib/openc3/models/interface_model.rb

```diff
@@ -165,6 +165,7 @@ def build
         klass = OpenC3.require_class(protocol[1])
         interface_or_router.add_protocol(klass, protocol[2..-1], protocol[0].upcase.intern)
       end
+      interface_or_router.start_raw_logging if @log_stream
```

Comment:
Raw logging is currently broken except for the APIs manually starting it. This allows it to start with the `LOG_RAW` or `LOG_STREAM` flag. But maybe this should move to the keyword parsing like in bridge_config?